packages/block-editor/src/components/block-list/block-mobile-toolbar.native.js

```diff
@@ -28,7 +28,7 @@ const BlockMobileToolbar = ( {
 
 		<View style={ styles.spacer } />
 
-		<InspectorControls.Slot />
+		<BlockSettingsButton.Slot />
```

Comment:
It's because we render the settings button `Fill` component inside Inspector Controls when it has any children. Using it in that way we can automatically show/hide setting button depending if block has some settings wrappen inside`InspectorControl` component inside `edit.native.js` file.

At this moment it was my best idea to make the API on mobile usage as much as possible to web.